Tools/peg_generator/scripts/grammar_grapher.py

```diff
@@ -91,11 +91,11 @@ def main() -> None:
         references[name] = set(references_for_item(rule))
 
     # Flatten the start node if has only a single reference
-    root_node = "start"
-    if start := references["start"]:
+    root_node = "file" # The start rule is called 'file' now
```

Comment:
What about if we add a new argument to allow to pass the starting rule (that defaults to `file`)? This way we don't need to have this hardcoded here and we could plot different starting nodes :)

Reply:
That's a good idea. Although I think the bigger question is, do we even need `grammar_grapher` in the upstream repo? I can't really see the use for it.

Reply:
I think is useful for visualizing the structure of the grammar or parts of it. I have used it two or three times to debug or understand stuff :)

Reply:
Oh, okay then. I'll work on it then. :)